src/pages/Home.js

```diff
@@ -83,26 +83,37 @@ function Header({ carePlan, patient }) {
   );
 }
 
-const HomeContent = ({ carePlan }) => {
-  const tabs = carePlan
-    ? [
-        {
+const getCarePlanTabs = (carePlan) => {
+  const tabOrder = process.env.REACT_APP_HOME_TAB_ORDER ?
+    process.env.REACT_APP_HOME_TAB_ORDER.split(',').map((tabName) => tabName.toLowerCase()) :
+    ["tasks", "goals", "detais"]
+  return tabOrder.map((tabName) => {
+    switch (tabName) {
```

Comment:
I'd store the `to` inside a variable like so
```suggestion
    const to = `/home/${carePlan.id}/${tabName}`
    switch (tabName) {
```